src/test/java/com/bootcamp/topic3/test/ShoppingCartServiceTest.java

```diff
@@ -0,0 +1,58 @@
+package com.bootcamp.topic3.test;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.bootcamp.topic3.cart.CartItem;
+import com.bootcamp.topic3.service.ShoppingCartService;
+import com.bootcamp.topic3.service.ShoppingCartServiceFactory;
+
+public class ShoppingCartServiceTest {
+	
+	private ShoppingCartService service;
+	private CartItem c2;
+	private CartItem c1;
+	
+	@Before
+	public void setUp(){
+		service = ShoppingCartServiceFactory.getShoppingCartService();
+
+		c1 = new CartItem("p1", 10, 2);
+		c2 = new CartItem("p2", 40, 3);
+	
+		service.addCartItem(c1);
+		service.addCartItem(c2);
+	}
+	
+	@After
+	public void after(){
+		service.clearCart();
+	}
+
+	@Test
```

Comment:
This Unit Test suite includes only possitive tests, what about the negative test? I mean, the cases of add and delete a null object is not covered.

Reply:
Hi I had some problems to understand how pull request works . Finally I did the corrections and created a diferent PR  https://github.com/cristian-rodrig/gitBootCamp/pull/4